junction/static/less/app.less

```diff
@@ -0,0 +1,10 @@
+@import "vendor.less";
+
+@import "variables.less";
+@import "bootstrap_overrides.less";
+
+@import "utils.less";
+@import "base.less";
+
+@import "editor.less";
+@import "home.less";
```

Comment:
Even `less` supports mixins & inheritance. I think `sass` supports more advanced things too but we will probably never require them, the only reason I chose LESS is because the official twitter bootstrap bower package supports LESS https://github.com/twbs/bootstrap & I guess python.org doesn't use bootstrap.